examples/Finding_Local_Peaks_in_Solar_Data.py

```diff
@@ -0,0 +1,76 @@
+"""
+=================================
+Finding Local Peaks in Solar Data
+=================================
+
+How to find regions of local maxima
+"""
+import astropy.units as u
+from skimage.feature import peak_local_max
+import matplotlib.pyplot as plt
+from mpl_toolkits import mplot3d
+from mpl_toolkits.mplot3d import Axes3D
+import numpy as np
+
+import sunpy.map
+from sunpy.data.sample import AIA_193_IMAGE
+
+###############################################################################
+# We will first create a Map using some sample data and display it.
+
+aiamap = sunpy.map.Map(AIA_193_IMAGE)
+plt.figure()
+aiamap.plot()
+plt.colorbar()
+
+
+###############################################################################
+# Before we find regions of local maxima, we need to create some variables that
+# store pixel coordinates for the 2D SDO/AIA data we are using.
+# These variables are used for plotting in 3D later on.
+
+x = y = np.arange(0, len(aiamap.data))
+X, Y = np.meshgrid(x, y)
+
+
+###############################################################################
+# We will only consider peaks within the AIA data that are above a threshold
+# value. Once these values are thresholded, the next step is to
```

Comment:
This comment is a little out of date, seeing how we are using a relative threshold now, it doesn't quite read right any more.

Reply:
@Cadair , so mentioning what `threshold_rel` does will do good I guess

Reply:
yeah I was thinking something along those lines.